product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -15,31 +15,31 @@ The two parts to EDB Postgres Distributed's proxy layer are:
 * Proxy configuration and routing information, which is maintained by the PGD consensus mechanism.
 * The PGD Proxy service, which is installed on a host. It connects to the PGD cluster, where it reads its configuration and listens for changes to the routing information.
 
-This is normally installed in highly available configuration (at least two instances of the proxy service per PGD group).
+This layer is normally installed in a highly available configuration (at least two instances of the proxy service per PGD group).
 
 Once configured, the PGD Proxy service monitors routing changes as decided by the EDB Postgres Distributed cluster. It acts on these changes to ensure that connections are consistently routed to the correct nodes.
 
 Configuration changes to the PGD Proxy service are made through the PGD cluster. 
 The PGD Proxy service reads its configuration from the PGD cluster, but the proxy service must be restarted to apply those changes.
 
 The information about currently selected write and read nodes is visible in
-`bdr.node_group_routing_summary`. This is node-local view: the proxy
+`bdr.node_group_routing_summary`. This is a node-local view: the proxy
 always reads from Raft leader to get a current and consistent view.
 
 ## Leader selection
 
 The write leader is selected by the current Raft leader (either subgroup one or top-level group one,
 depending on whether the leader for the subgroup or the cluster's top-level group is being selected).
```

Comment:
I was trying to break this into two sentences but I was having a hard time telling what the parenthetical was referring to. Would this be correct? 

The write leader is selected by the current Raft leader. The Raft leader is either subgroup one or top-level group one,
depending on whether the leader for the subgroup or the cluster's top-level group is being selected.

Reply:
The write lead is selected by the current Raft leader for the group the proxy is part of. This could be the Raft leader for a subgroup or leader for the entire cluster.